Drivers/Periphs/Src/RTC.c

```diff
@@ -0,0 +1,41 @@
+//The Real Time Clock (RTC) communicates with the uC over I2C2
+#include "RTC.h"
+
+RTCData_t *RTCData;
+HAL_StatusTypeDef error;
+
+/**
+ * Sets Real-Time Clock to 12 hour mode (this can be changed to 24 hour mode instead if desired)
+ * @param data buffer that will hold data 
+**/
+ErrorStatus RTC_Init(RTCData_t *data){
+    //configure RTC
+    uint8_t config[2];
+    config[0] = CONTROL_1; //register address
+    config[1] = 0x08; //data to be transmitted 
+    HAL_I2C_Master_Transmit_IT(&hi2c2, DEVICE_ADDRESS, config, 2); // select 12 hour mode 
```

Comment:
Shouldn't error be set equal to the output of this function? Same for the rest.

Reply:
fixed